src/org/zaproxy/zap/extension/pscanrules/TestInfoPrivateAddressDisclosure.java

```diff
@@ -1,153 +1,159 @@
-/*

```

Comment:
I have no idea what's wrong here. 
The Eclipse-diff shows me just the 16 edited lines.
Copied the files from master and  issue-2834... to another folder and a comparison with Notepad++ also shows just the really changed lines.
Both files have CRLFs, and are UTF-8 (which shouldn't matter here)
But interesting, that this is the only file with that problem in GitHub-diff